README.md

```diff
@@ -1,14 +1,14 @@
 <a name="eslint-plugin-jsdoc"></a>
-# eslint-plugin-jsdoc
-
-[![GitSpo Mentions](https://gitspo.com/badges/mentions/gajus/eslint-plugin-jsdoc?style=flat-square)](https://gitspo.com/mentions/gajus/eslint-plugin-jsdoc)
-[![NPM version](https://img.shields.io/npm/v/eslint-plugin-jsdoc.svg?style=flat-square)](https://www.npmjs.org/package/eslint-plugin-jsdoc)
-[![Travis build status](https://img.shields.io/travis/gajus/eslint-plugin-jsdoc/master.svg?style=flat-square)](https://travis-ci.org/gajus/eslint-plugin-jsdoc)
-[![js-canonical-style](https://img.shields.io/badge/code%20style-canonical-blue.svg?style=flat-square)](https://github.com/gajus/canonical)
-[![Discord Chat](https://img.shields.io/badge/chat-on%20disord-green.svg?logo=discord)](https://discord.gg/kFFy3nc)
-
-JSDoc linting rules for ESLint.
-
+# eslint-plugin-jsdoc
+

```

Comment:
Lots of newline changes here. If I keep `\n` as it was, it doesn't allow me to push, asking that I run `npm run create-readme`. Running that changes all instances to `\r\n`. I suppose the hook doesn't work too well on Windows...

Reply:
Hmm--I've amended it, but as far as fixing the root problem, I'm not sure. Did you have the source README open in your editor? If so, I'd think that could do it, though I'd expect most editors would be aware of `.editorconfig`.